pom.xml

```
@@ -83,6 +83,7 @@
         <module>test-utils/testcontainers</module>
         <module>test-utils/junit5-extension</module>
         <module>test-utils/assert/core</module>
+        <module>test-utils/assert/qa</module>
```

```suggestion
        <module>test-utils/assert</module>
```
```suggestion
        <module>test-utils/assert/core</module>
        <module>test-utils/assert</module>
```

With introducing the parent `pom.xml`, we don't need to call each child module separately, right? We probably need to change it for the sonatype profile as well.

---

Oh, yes, I forgot about that. But do we want the QA module to be released? Wouldn't it make more sense to only release core?

---

We need to call `test-utils/assert` in the top-level reactor. Otherwise, building the child module might fail since the root `pom.xml` cannot be found.

To avoid the QA module being pushed to maven central, we could do something like this in the root `pom.xml`:
```xml
  <profiles>
    <profile>
      <id>distro</id>
      <activation>
        <activeByDefault>true</activeByDefault>
      </activation>
      <modules>
        <module>core</module>
        <module>qa</module>
      </modules>
    </profile>

    <profile>
      <id>sonatype-oss-release-12</id>
      <modules>
        <module>core</module>
      </modules>
    </profile>
  </profiles>
```

---

Makes sense. I will add the profiles and change the platform root pom.